artifacts/definitions/Windows/Registry/WDigest.yaml

```diff
@@ -0,0 +1,55 @@
+name: Windows.Registry.WDigest
```

Comment:
This one is cool but I wonder if it falls into the bigger "auditing" kind of artifact? We can certainly check a large number of keys/values at the same time for different reasons and people would probably run them all the same time.

We can add it for now but will be looking to consolidate the similar checks into one artifact.